google-cloud-clients/pom.xml

```diff
@@ -167,6 +167,7 @@
     <!-- test dependency versions -->
     <easymock.version>3.4</easymock.version>
     <objenesis.version>2.6</objenesis.version>
+    <opencensus.version>0.16.0</opencensus.version>
```

Comment:
There is already an `<opencensus.version>` above.  Please remove this duplicate entry.